clouddriver-core/src/test/groovy/com/netflix/spinnaker/clouddriver/orchestration/DefaultOrchestrationProcessorSpec.groovy

```diff
@@ -108,11 +108,12 @@ class DefaultOrchestrationProcessorSpec extends Specification {
     where:
     exception                     | sagaId               || retryable
     new RuntimeException()        | null                 || false
-    new NonRetryableException()   | null                 || false
-    new RuntimeException()        | new SagaId("a", "a") || true
+    new RetryableException()      | null                 || false
+    new RuntimeException()        | new SagaId("a", "a") || false
     new NonRetryableException()   | new SagaId("a", "a") || false
-    new SomeDynamicException()    | new SagaId("a", "a") || false
-    new AnotherDynamicException() | new SagaId("a", "a") || false
+    new RetryableException()      | new SagaId("a", "a") || true
+    new SomeDynamicException()    | new SagaId("a", "a") || true
```

Comment:
im confused... how come `SomeDynamic` and `AnotherDynamic` are retryable now?

Reply:
I reversed the logic in `ExceptionClassifier` so now we opt-in to retrying these exceptions.